brainiak/reprsimil/brsa.py

```diff
@@ -254,20 +252,22 @@ class BRSA(BaseEstimator, TransformerMixin):
         GP prior on log(SNR), and returns the MAP estimate of tau^2.
         It can be either prior_GP_var_inv_gamma for inverse-Gamma
         or prior_GP_var_half_cauchy for half-Cauchy.
-        But half-Cauchy prior is actually imposed on tau.
-        tau_range still describes the range of tau in the prior
-        in both cases. Inverse-Gamma prior
-        penalizes for very small tau and very large tau,
-        while half-Cauchy prior only penalizes for very large tau.
-        Both the functions are part of brsa module, which you need
-        to import together with BRSA in order to use.
-        For example::
-        from brainiak.reprsimil.brsa import BRSA, prior_GP_var_inv_gamma
-        brsa = BRSA(tau2_prior=prior_GP_var_inv_gamma)
-        Or, if you do are happy with the default inverse-Gamma prior,
-        then you can ignore this argument::
-        from brainiak.reprsimil.brsa import BRSA
-        brsa = BRSA()
+        half-Cauchy prior is in fact imposed on tau.
+        But tau_range describes the range of tau in the prior in both cases.
+        Both functions are part of brsa module.
+        See also `.prior_GP_var_inv_gamma` and
+        `.prior_GP_var_half_cauchy`
+        To use the default inverse-Gamma prior, you can ignore this argument::
+
+            from brainiak.reprsimil.brsa import BRSA
+            brsa = BRSA()
+
+        If you want to try the alternative half-Cauchy prior,
+        then you need to import it in addition to BRSA::
+
+            from brainiak.reprsimil.brsa import BRSA, prior_GP_var_half_cauchy
+            brsa = BRSA(tau2_prior=prior_GP_var_half_cauchy)
+
```

Comment:
You must also add the functions to `__all__` to make them part of the API. This will also add them to the HTML docs.